documentation/learningPath/aks.md

```diff
@@ -2,3 +2,82 @@
 ### Was this documentation helpful? [Share feedback](https://www.research.net/r/DGDQWXH?src=documentation%2FlearningPath%2Faks)
 
 # AKS
+
+In addition to its availability as a .NET CLI tool, the `dotnet monitor` tool is available as a prebuilt Docker image that can be run in container runtimes and orchestrators, such as Kubernetes. To learn more about using `dotnet monitor` with Kubernetes, you can check out our [example deployment](https://github.com/dotnet/dotnet-monitor/blob/main/documentation/kubernetes.md) and our [video tutorial](https://github.com/dotnet/dotnet-monitor/tree/main/samples/AKS_Tutorial). This section covers how to test your development version of `dotnet monitor` in AKS; however, we recommend having some basic experience with AKS before following this workflow.
+
+## What This Workflow Does
+
+This workflow takes your local development copy of `dotnet-monitor`, patches it with a local development copy of the [.NET Core Diagnostics Repo](https://github.com/dotnet/diagnostics#net-core-diagnostics-repo), and makes it available as an image for you to consume in an ACR (Azure Container Registry). Note that there are many other ways to do this - this is meant to serve as a basic template that can be adapted to match your needs.
+
+1. Open `pwsh` and run the [generate-dev-sln script](https://github.com/dotnet/dotnet-monitor/blob/main/generate-dev-sln.ps1), providing a path to your local copy of the diagnostics repo.
+
+>**Note**: If your changes do not involve the [.NET Core Diagnostics Repo](https://github.com/dotnet/diagnostics#net-core-diagnostics-repo), you don't need to complete this step.
+
+```bash
+cd C:\your-path\dotnet-monitor 
+.\generate-dev-sln.ps1 C:\your-path\diagnostics
+```
+
+2. Publish `dotnet monitor` to your desired (local) target location (TEMP is used throughout this example)
+
+```bash
+dotnet publish .\src\Tools\dotnet-monitor -o $env:TEMP\dotnet-monitor -c Release -f net8.0
+```
+
+3. Add a Dockerfile to the `dotnet monitor` directory created in the previous step. Below is an example of the contents of the Dockerfile:
+
+```bash
+ARG REPO=mcr.microsoft.com/dotnet/aspnet
+FROM $REPO:7.0.3-alpine3.17-amd64
```

Comment:
```suggestion
FROM $REPO:8.0-alpine-amd64
```
Couple of points:
- The image target framework needs to match whichever target framework you publish for (above, you are publishing `net8.0`)
- Use the major.minor and versionless distro tag so that you get the latest supported image.
- Alpine images are not actually something we are offering for .NET 8, but it should still work.
- Add an instruction between #2 and #3 to pull the base image (e.g. `docker pull mcr.microsoft.com/dotnet/aspnet:8.0-alpine-amd64`) so you are not using a cached image that might be vulnerable.